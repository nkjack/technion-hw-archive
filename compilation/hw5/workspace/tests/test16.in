//arithmetic
void main() {
	printi(8);
	print("\n");
	printi(150b);
	print("\n");
	printi(1+2+3+1);
	print("\n");
	printi(120b-50b-1b-7b);
	print("\n");
	printi(10-2*3+1);
	print("\n");
	printi(1338-8);
	print("\n");
	printi(10/2);
	print("\n");
	printi(1+400/3b);
	print("\n");
}