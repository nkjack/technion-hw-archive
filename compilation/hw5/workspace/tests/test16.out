Loaded: ./exceptions.s
8
150
7
62
5
1330
5
134
